M_K.H

```diff
@@ -12,6 +12,7 @@ int mk_getextchar(void);
 void mk_emptychar(void);
 int mk_kbhit(void);
 int mk_getstate(int kod);
+int controlaltnyomva(void);
```

Comment:
I don't understand how this relates to cleanup https://github.com/elmadev/elma-classic/pull/91/commits/35d5e18839c3f48c718f91e408c1350b83400d74